app/assets/stylesheets/rails_views/marriage.scss

```diff
@@ -9,7 +9,7 @@ form.button_to {
     font-family: inherit;
     font-size: inherit;
     background: none;
-    color: rgb(0, 0, 238);
+    color: rgb(37, 99, 235);
```

Comment:
Mostly unrelated change to match this color (for "Delete") to the color provided by Tailwind for "Edit".